### PR TITLE
Aw/pdf feedback

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     .mb-2
       %dt.inline Case Worker:
       %dd.inline= @assessment.user&.name
-  - if @client.patient.current_qa_factory.ca_completed_qa.present?
+  - if true # @client.patient.current_qa_factory.ca_completed_qa.present?
```
Comment (eanders) [inline drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml:16]: Should there be a comment on the why this is always true?  Or maybe not an if?

Comment (awisema) [reply drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml:16]: Debug code that crept it -- I just fixed it.

Comment (eanders) [reply drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml:16]: ok, not seeing that change yet...

Comment (awisema) [reply drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml:16]: Went straight to the staging branch -- I thought you were out today, so, I didn't wait to merge :-)

**Review by eanders — COMMENTED** [review]
Looks good

**Pull request merged**